### DISCUSSION TITLE
Mark as favorite during scan?

### DISCUSSION THREAD

**Discussion by larsxschneider** [photoview/photoview#738] [Q&A and Troubleshooting]
Is there currently a way to set the "favorite" state during scan somehow?

Thanks you for this amazing tool @viktorstrate 🙇

**Comment by viktorstrate** [comment]
It doesn't work to click the heart icon on the thumbnail in the same way as after the scanner has finished?

**Reply by larsxschneider** [reply]
I can _manually_ mark an image as favorite but I want to do it automatically on import.

**Background:**
I use [PhotoSync](https://www.photosync-app.com/home.html) to transfer photos from my iPhone to my NAS. On transfer I rename the files and add a ❤️ to all files that I have already liked on the iPhone. I would like to parse the ❤️ on Photoview import to see the images that I have liked on the iPhone in Photoview too.
